RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -1252,38 +1271,142 @@ vector<TransientVertex> DAClusterizerInZ_vect::vertices_in_blocks(const vector<r
 #endif
 
   vector<TransientVertex> clusters;
+  if (nv == 0) {
+    return clusters;
+  }
 
   GlobalError dummyError(0.01, 0, 0.01, 0., 0., 0.01);
+  vector<reco::TransientTrack> vertexTracks;  // moved here from inside the following loop
+
   for (unsigned int k = 0; k < nv; k++) {
     if (!vtx_track_indices[k].empty()) {
-      GlobalPoint pos(0, 0, vertices_tot[k].first);
-      vector<reco::TransientTrack> vertexTracks;
+      // GlobalPoint pos(0, 0, vertices_tot[k].first); //moved
```

Comment:
do we need to keep this commented code here?

Reply:
looks like a remnant from development, indeed